workflow/Snakefile

```diff
@@ -32,6 +34,8 @@ config = make_paths_absolute(config, os.getcwd())
 config["results_dir"] = os.path.join(os.getcwd(), config["results_dir"])
 RESULTS_DIR = config["results_dir"]
 E2G_DIR_PATH = config["E2G_DIR_PATH"]
+MODEL_DIR = config["model_dir"]
+include: os.path.join(E2G_DIR_PATH, "workflow", "rules", "utils.smk")
```

Comment:
if we do this, then we can utilize the existing `make_paths_absolute` fn defined here.